CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## Unreleased
+## 1.6.2 - 1.8.1
```

Comment:
Imo, these should be separated. We have had 3 new versions since 1.6.2, but there are only 2 items in the CHANGELOG.md. Could you fill the missing changes?